lib/run.go

```diff
@@ -56,6 +62,10 @@ func (a *ACBuild) Run(cmd []string, workingdir string, insecure bool) (err error
 		return fmt.Errorf("the run subcommand must be run as root")
```

Comment:
Can we push this check into the engine now..? or are there other things I'm not thinking about